### PR TITLE
[Abyssea] Set event 2180 when timer is over and move set pos in onEventFinish

### PR DESCRIPTION
**Pull request by Chronos77**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Set event 2180 when timer is over and move set pos in onEventFinish

review of the time display at the end of the timer (see it on retail)

## Steps to test these changes

Enter in abyssea and do not take visitant status 

Wait 5 minutes

### CODE REVIEW

File: scripts/globals/abyssea.lua [hunk 1]
```diff
@@ -1098,6 +1098,13 @@ xi.abyssea.onZoneIn = function(player)
     end
 end
 
+xi.abyssea.onEventFinish = function(player, csid, option)
```
Comment (claywar) [inline scripts/globals/abyssea.lua:1101]: onEventFinish (even if not used) should have 4 parameters (player, csid, option, npc)

File: scripts/globals/effects/visitant.lua [hunk 2]
```diff
@@ -6,7 +6,7 @@ require("scripts/globals/zone")
 -----------------------------------
 local effectObject = {}
 
-local remainingTimeLimits = { 300, 120, 60, 30, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1 }
+local remainingTimeLimits = { 300, 240, 180, 120, 60, 30, 10, 5, 4, 3, 2, 1 }
```
Comment (claywar) [inline scripts/globals/effects/visitant.lua:9]: Can you confirm that this is the behavior on retail for updates?  In original testing, the previous version was what I observed.

Comment (Chronos77) [reply scripts/globals/effects/visitant.lua:9]: image
This is a capture i made on retail 10 minutes ago

File: scripts/globals/abyssea.lua (hunk 1 above)
Comment (Chronos77) [reply scripts/globals/abyssea.lua:1101]: same for zone object? 

this is an example for AlTaieu
image

however, the name may not be correct, could I put onZoneEventFinish?

Comment (claywar) [reply scripts/globals/abyssea.lua:1101]: Fair enough, since there is no npc to interact with here

**Review by claywar — APPROVED** [review]

File: scripts/globals/abyssea.lua (hunk 1 above)
Comment (TeoTwawki) [reply scripts/globals/abyssea.lua:1101]: is actually the event target, which is some edge cases is not an npc. on occasion it is the player themself. I don't think the zone object is ever it tho. On retail SE usualy uses a stand in dummy NPC for such interactions.

**Pull request merged**